config/crd/bases/nfd.kubernetes.io_nodefeaturediscoveries.yaml

```diff
@@ -45,6 +44,11 @@ spec:
                   type: string
                 nullable: true
                 type: array
+              grpcMode:
+                description: GrpcMode defines whether the NFD-master should be deployed
+                  in gRPC mode or not. If enabled, the Operator will deploy an NFD-Master
+                  in gRPC mode that will listen for incoming gRPC requests.
```

Comment:
Should we describe what happens if gRPC mode is disabled? I.e. the NodeFeature CRD API is disabled.